spec/spec.json

```diff
@@ -98,7 +98,8 @@
 		{ "name": "plural", "type": "string" },
 		{ "name": "description", "type": "string", "required": false },
 		{ "name": "deprecation", "type": "deprecation", "required": false },
-		{ "name": "values", "type": "[enum_value]" }
+		{ "name": "values", "type": "[enum_value]" },
+		{ "name": "attributes", "type": "[attribute]", "default": "[]" }
```

Comment:
I think the only reason to do optional would be to indicate that somebody explicitly decided that there would be no attributes. Personally for this type of data my preference would be to return an empty list - then implementers can always just filter the collection (vs checking for empty)... and in other languages helps to avoid null pointers. So my preference would be to leave as a default []